src/autoscaler/healthendpoint/server.go

```diff
@@ -20,6 +20,13 @@ import (
 	"golang.org/x/crypto/bcrypt"
 )
 
+const (
+	LIVELINESS_PATH string = "/health/liveliness"
```

Comment:
The endpoint name look's better -> /health/liveness

Inspiration https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/

Reply:
Lets do that change!